firestore/query.go

```diff
@@ -569,8 +579,48 @@ func trunc32(i int) int32 {
 	return int32(i)
 }
 
+// Get returns an array of query's resulting documents.
+func (q Query) Get(ctx context.Context) ([]*DocumentSnapshot, error) {
+	limitedToLast := q.limitToLast
+
+	if q.limitToLast {
+		// Flip order statements before posting a request.
+		for i := range q.orders {
+			if q.orders[i].dir == Asc {
+				q.orders[i].dir = Desc
+			} else {
+				q.orders[i].dir = Asc
+			}
+		}
+		// Swap cursors.
+		q.startVals, q.endVals = q.endVals, q.startVals
+		q.startDoc, q.endDoc = q.endDoc, q.startDoc
+		q.startBefore, q.endBefore = q.endBefore, q.startBefore
+
+		q.limitToLast = false
+	}
+	docs, err := q.Documents(ctx).GetAll()
+	if err != nil {
+		return nil, err
+	}
+	if limitedToLast {
+		// Flip docs order before return.
+		for i, j := 0, len(docs)-1; i < j; {
+			docs[i], docs[j] = docs[j], docs[i]
+			i++
+			j--
+		}
+	}
+	return docs, nil
+}
+
 // Documents returns an iterator over the query's resulting documents.
 func (q Query) Documents(ctx context.Context) *DocumentIterator {
+	if q.limitToLast {
+		return &DocumentIterator{
+			err: errors.New("Query results for queries that include limitToLast constraints cannot be streamed. Use Query.Get() instead."),
```

Comment:
Can we prefix the error with package name. i.e. `"firestore: queries that include ..."`

Reply:
Done